ironfish/src/rpc/clients/tcpClient.ts

```diff
@@ -23,8 +23,13 @@ export class RpcTcpClient extends RpcSocketClient {
   connection: RpcClientConnectionInfo
   private messageBuffer: MessageBuffer
 
-  constructor(host: string, port: number, logger: Logger = createRootLogger()) {
-    super(logger.withTag('tcpclient'))
+  constructor(
+    host: string,
+    port: number,
+    rpcAuthTokoen: string,
```

Comment:
This is a typo, and I don't think it should be required since auth is not required. I also think we can just name this `authToken` since this is already in the RpcClient.

```suggestion
   authToken: string,
```